apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -990,4 +999,79 @@ export class BookingsService_2024_08_13 {
       t: await getTranslation("en", "common"),
     });
   }
+
+  async updateBooking(bookingUid: string, body: UpdateBookingInput_2024_08_13) {
+    const existingBooking = await this.bookingsRepository.getByUid(bookingUid);
+    if (!existingBooking) {
+      throw new NotFoundException(`Booking with uid=${bookingUid} not found`);
+    }
+    const { location } = body;
+
+    if (!location) {
+      return this.getBooking(existingBooking.uid);
+    }
+
+    return await this.updateBookingLocation(existingBooking, location);
```

Comment:
Like I wrote in the previous review comment - I think it is better to check if location is passed and then call this.updateBookingLocation and by default simply return this.getBooking(existingBooking.uid); - this way next time we want to update for example attendees we just need to add a new attendee update section instead of having to also update `if (!location && !attendee)` etc